src/storage/slab/slab.h

```diff
@@ -105,6 +107,7 @@ struct slab {
     rel_time_t        utime;        /* last update time in secs */
     uint8_t           id;           /* slabclass id */
     uint32_t          padding:24;   /* unused */
+    uint32_t          refcount;     /* number of reserved items */
```

Comment:
I don't think this description is clear, it makes me thing that each slab reserves and not that it is itself reserved